pkg/phlaredb/profile_test.go

```diff
@@ -279,3 +279,67 @@ func TestProfileIndex_Add_OutOfOrder(t *testing.T) {
 	require.Equal(t, []int64{20, 50, 80, 100, 110}, tsOrder)
 
 }
+
+func Test_rowRangesWithSeriesIndex_getSeriesIndex(t *testing.T) {
+	testCases := []struct {
+		name        string
+		ranges      rowRangesWithSeriesIndex
+		rowNum      int64
+		searchHint  int
+		expectIdx   int
```

Comment:
we should rename this to  expectedSeriesIndex, to not confuse with any other indexes